duniverse/mdx/CHANGES.md

```diff
@@ -2,30 +2,22 @@
 
 #### Added
 
-#### Changed
```

Comment:
why did you change that the beginning of that file?

Reply:
Same as above